jenkins-pipelines/oss/raft/longevity-5000-tables.jenkinsfile

```diff
@@ -8,4 +8,6 @@ longevityPipeline(
     region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_user_batch_custom_time',
     test_config: '''["test-cases/scale/longevity-5000-tables.yaml", "configurations/raft/enable_raft_experimental.yaml"]''',
+
+    timeout: [time: 4440, unit: 'MINUTES'],
```

Comment:
It is removed in the first commit and added back in the second.
Both commits should not have changed this file.